src/App.tsx

```diff
@@ -1,14 +1,63 @@
-import React from 'react';
+import React, {
+  useCallback, useMemo, useRef, useState,
+} from 'react';
+
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { People } from './components/People';
+
+type Props = {
+  delay: number
+};
+
+export const App: React.FC<Props> = ({ delay }) => {
+  const [user, setUser] = useState('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
```
it'll be more accurate